src/l3.c

```diff
@@ -37,14 +37,16 @@ struct l3_entry
 /**
  * L3 Log Structure definitions:
  */
-struct l3_log
+typedef struct l3_log
 {
     uint64_t idx;
     uint64_t fbase_addr;
     uint64_t pad0;
     uint64_t pad1;
     struct l3_entry slots[L3_MAX_SLOTS];
-} *l3_log;
+} L3_LOG;
+
+L3_LOG *l3_log;
```

Comment:
Are you ok with this naming change?

I think typedef'ed `L3_LOG` for common structure is a pretty normal convention.

I can imagine that as this tooling grows, this L3_LOG may creep into other files, so it's better we finalize and resolve the naming choice, if you were to find this objectionable.